src/LookupModel.php

```diff
@@ -57,7 +57,7 @@ public function __construct(array $lookup)
         $this->latitude = $this->getOrNull('latitude', $lookup);
         $this->longitude = $this->getOrNull('longitude', $lookup);
         $timezone = $this->getOrNull('timezone', $lookup);
-        if (!is_null($timezone)) {
+        if (!is_null($timezone) && in_array($timezone, timezone_identifiers_list())) {
```

Comment:
I've not seen an invalid timezone other than empty or 'n/a' since I've been using your service, but I don't necessarily like making the assumption that those are the only invalid ones.  

I'll yield to you guys on that, I agree it depends on what your data looks like.